depfinder/main.py

```diff
@@ -70,11 +70,24 @@
     Loader=yaml.SafeLoader,
 )
 
+with open('depfinder/pkg_data/name_mapping.yaml', 'r') as f:
+   mapping_list = yaml.load(f, Loader= yaml.SafeLoader)
 
-def _split(name):
-    named_space = pkg_data['_NAMEDSPACE_MAPPING'].get(name)
-    return named_space if named_space else name.split('.')[0]
+namespace_mapping = {}
+for pkg in mapping_list:
+   if pkg['import_name'] != pkg['conda_name']:
```

Comment:
I'm not certain if this is correct. `ruamel.yaml` is the name of the import and the conda name, but it is a namespace package. You may need to check if the import name has `.` instead.